CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## TBD
+
+### Bug fixes
+
+* Allow the metadata in leaveBreadcrumb to be null rather than enforcing non-null , aligning bugsnag-android with our other SDKs
```

Comment:
```suggestion
* Allow the metadata in `leaveBreadcrumb` to be null rather than enforcing non-null, aligning `bugsnag-android` with our other SDKs
```
this entry also needs a PR link